packages/react-components/global-context/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "peerDependencies": {
     "@types/react": ">=16.14.0 <19.0.0",
-    "@types/react-dom": ">=16.14.0 <19.0.0",
+    "@types/react-dom": ">=16.9.0 <19.0.0",
```

Comment:
Just curious, why not `>=16.9.24 <19.0.0`?

Reply:
none of the other deps used the last value, so I assumed it was convection. react types tops out at 16.14.55